float64[] ang
float64[] med
float64[] med_f
float64[] pos_person_x
float64[] pos_person_y
